third_party/move/move-compiler-v2/transactional-tests/tests/operator_eval/underscore.operator-eval-lang-1.exp

```diff
@@ -0,0 +1,173 @@
+processed 13 tasks
+
+task 0 'publish'. lines 1-77:
+Error: compilation errors:
+ error: undeclared `_`
+   ┌─ TEMPFILE:32:9
+   │
+32 │         _  // undefined
+   │         ^
+
+error: duplicate declaration of `_`
+   ┌─ TEMPFILE:34:30
+   │
+34 │     public fun test8(_: u64, _: u64): u64 {
+   │                      -       ^
+   │                      │        
+   │                      previous declaration of `_`
+
+error: duplicate declaration of `_`
+   ┌─ TEMPFILE:38:37
+   │
+38 │     inline fun fun9(x: u64, _: u64, _: u64): u64 {
+   │                             -       ^
+   │                             │        
+   │                             previous declaration of `_`
+
+error: duplicate declaration of `_`
+   ┌─ TEMPFILE:44:38
+   │
+44 │     inline fun fun10(x: u64, _: u64, _: |u64|u64): u64 {
+   │                              -       ^
+   │                              │        
+   │                              previous declaration of `_`
+
+error: cannot pass `integer` to a function which expects argument of type `|u64|u64`
+   ┌─ TEMPFILE:48:21
+   │
+48 │         fun10(4, 3, 2)
+   │                     ^
+
+
+
+task 1 'run'. lines 79-79:
+Error: Function execution failed with VMError: {
+    message: Linker Error: Module 0000000000000000000000000000000000000000000000000000000000c0ffee::m doesn't exist,
```

Comment:
Kind of noisy unrelated followup errors.

Reply:
Its now a regression test.  No need to ever look at it again, unless it changes.